specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2025-03-01/elasticsan.json

```diff
@@ -1570,6 +1570,138 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ElasticSan/elasticSans/{elasticSanName}/volumegroups/{volumeGroupName}/preBackup": {
+      "post": {
+        "tags": [
+          "VolumesBackupValidation"
+        ],
+        "description": "Validate whether a disk snapshot backup can be taken for list of volumes.",
+        "operationId": "VolumeGroups_PreBackup",
```

Comment:
As we discussed , please give accurate operationId for both APIs. 
PSH/CLI/SDK API name all come from this.